Gemfile

```diff
@@ -3,3 +3,4 @@ source "https://rubygems.org"
 gem "twitter"
 gem "trollop"
 gem "dotenv"
+gem "highline"
```

Comment:
I used [Highline#agree](https://github.com/JEG2/highline/blob/master/lib/highline.rb#L191) for the [prompt](https://github.com/mikemcquaid/TwitterDelete/pull/5/files#diff-5fe53d53ea29c15e6ef1bf2a468c4ce9R93)